support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -155,8 +155,16 @@ echo "$dest_dir" > $dest_dir_file
 # Create single_file file in plain text
 echo "$filename" > $filename_file
 
+set +e;
```

Comment:
Disabling exit on error is now unnecessary, as you are capturing the command error with a pipe ( `|| ...`)